modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -2975,6 +2976,84 @@ void ONNXImporter::parseCumSum(LayerParams& layerParams, const opencv_onnx::Node
     addLayer(layerParams, node_proto);
 }
 
+void ONNXImporter::parseDepthToSpace(LayerParams& layerParams, const opencv_onnx::NodeProto& node_proto_)
+{
+    // We parse "DepthToSpace" and "SpaceToDepth" in this function.
+    opencv_onnx::NodeProto node_proto = node_proto_;
+    const std::string& layer_type = node_proto.op_type();
+    CV_Assert(layer_type == "DepthToSpace" || layer_type == "SpaceToDepth");
+    layerParams.type = layer_type;
```

Comment:
> layerParams.type = layer_type;

Is it used anywhere?
Below its value is rewritten.

Reply:
Thanks for code reviewing. All issues have been fixed.